README.md

```diff
@@ -178,6 +167,8 @@ This is particularly important when there are massive volumes of incoming data/s
 
 ## Training: Start
 
+With a few lines of code, you can add ONNX Runtime into your existing training scripts and start seeing acceleration. The current preview version supports training acceleration for PyTorch transformer models on NVIDIA GPUs.
+
 **[End-to-End Sample Notebook]()**
```

Comment:
If there's no sample notebook, should this line be removed?

Reply:
it'll be added soon. placeholder.